### DISCUSSION TITLE
conjugation

### DISCUSSION THREAD

**Discussion by triadne** [birchill/10ten-ja-reader#1237] [Q&A] [answered]
can we have a little info about this? I often see that green line and I am unclear how to read that and therefore how to conjugate correctly.

**Comment by birtles** [comment]
Sure, I'm glad to provide extra info. Just to check, are you referring to these kind of annotations?

![image](https://github.com/birchill/10ten-ja-reader/assets/1232595/2b36b42d-8481-4b48-8ca4-1aaf1b86bc05)

**Reply by triadne** [reply]
Thanks for helping, I understand the part in red, but it's the brown dotted line that I don't understand. in this example, I think SU is being separated, so i would say this HANASERU... but the green line also separates HA.  I haven't found any place where these markings are explained.

**Reply by SaltfishAmi** [reply, marked as answer]
These dotted lines indicate pitch accent, i.e. which syllables are read high and which are read low.

**Reply by triadne** [reply]
OH!!! wow, I see... so it's not about conjugation at all! hmm.. I guess I have to find another way to learn conjugation better. OK well that clears it up, thank you!

**Reply by triadne** [reply]
would it be okay if I ask this then... in this case, it highlights the kanji and the hiragana, but what it displays is different, it shows ATSUI, but how do I say this then, ATSUIKU or ATSUKU? I'm not sure what the dictionary is saying to me.
![image](https://github.com/birchill/10ten-ja-reader/assets/29523389/03abadd2-218b-48fe-9dad-208084d78eab)

**Reply by birtles** [reply]
Oh, interesting. Our dictionary data doesn't split the stem from the inflected part but I think some dictionaries do, e.g. `あつ.い`.

I think we probably could add some text next to the `(< adv)` annotation applying the most common reading to the source text, e.g. `暑(あつ)く` with the あつ as ruby.

**Reply by triadne** [reply]
That sounds really helpful!! Thanks for reading my thoughts and whatever you decide to do :)
